content/docs/release-notes/22-10-0/feature-state.md

```diff
@@ -0,0 +1,560 @@
+---
+linkTitle: Feature grid
+title: 22.10.0 Feature Grid snapshot
+description: Snapshot of the Open 3D Engine (O3DE) feature status grid for the 22.10.0 release.
+toc: true
+---
+This feature grid shows the state of each feature system within **Open 3D Engine (O3DE)** for the 22.10.0 release. For an up-to-date feature grid and full notes, see the [o3de/community feature grid](https://o3de.github.io/community/features/form.html).
```

Comment:
Perhaps the link should point to https://o3de.github.io/community/features/ where notes regarding the feature grid are found.  An up-to-date feature grid can't be found at either URL, consider removing that part of the line.

Reply:
Updated the first two paragraphs, so it includes a link to the actual feature state form, as well as the guidelines. 

> This feature grid shows the state of each feature system within **Open 3D Engine (O3DE)** for the 22.10.0 release. For an up-to-date feature grid and full notes, refer to the [Feature State Form](https://o3de.github.io/community/features/form.html).
>
> Features are reported by each individual O3DE Special Interest Group (SIG), as outlined in the [O3DE SIG Features Editing tool](https://o3de.github.io/community/features/) guide. For more information about each SIG and their responsibilities, refer to the [o3de/community repository](https://github.com/o3de/community/) on GitHub.